monoio/src/buf/io_vec_buf.rs

```diff
@@ -175,11 +175,11 @@ unsafe impl IoVecBufMut for VecBuf {
         for (idx, iovec) in self.iovecs.iter_mut().enumerate() {
             if iovec.iov_len <= len {
                 // set_init all
-                self.raw[idx].set_len(iovec.iov_len);
+                self.raw[idx].set_init(iovec.iov_len);
                 len -= iovec.iov_len;
             } else {
                 if len > 0 {
```

Comment:
Shouldn't we set to 0 even if len is 0?

Reply:
IO ops from kernel and return zero length means EOF